packages/office-ui-fabric-react/src/components/HoverCard/HoverCard.types.ts

```diff
@@ -58,46 +54,115 @@ export interface IHoverCardProps extends React.HTMLAttributes<HTMLDivElement | H
 
   /**
-   * Optional target element to tag hover card on
+   * Callback when card hides
    */
-  target?: HTMLElement | string;
+  onCardHide?: () => void;
 
   /**
-   * Callback when card becomes visible
+   * HotKey used for opening the HoverCard when tabbed to target.
+   * @default 'KeyCodes.c'
    */
-  onCardVisible?: () => void;
+  openHotKey?: KeyCodes;
 
   /**
-   * Callback when card hides
+   * Additional PlainCard props to pass through HoverCard like renderers, target, gapSpace etc.
+   * Used along with 'type' prop set to HoverCardType.basic.
+   * See for more details ICardProps and IPlainCardProps interfaces.
    */
-  onCardHide?: () => void;
+  plainCardProps?: IPlainCardProps;
 
   /**
-   * Trap focus or not
+   * Whether or not to mark the container as described by the hover card.
+   * If not specified, the caller should mark as element as described by the hover card id.
    */
-  trapFocus?: boolean;
+  setAriaDescribedBy?: boolean;
+
+  /**
+   * Set to true to set focus on the first focusable element in the card. Works in pair with the 'trapFocus' prop.
+   * @default false
+   */
+  setInitialFocus?: boolean;
 
   /**
    * Should block hover card or not
    */
   shouldBlockHoverCard?: () => void;
 
   /**
-   * Set first focus into hover card.
+   * If true disables Card dismiss upon mouse leave, so that card sticks around.
    * @default false
    */
-  setInitialFocus?: boolean;
+  sticky?: boolean;
 
   /**
-   * HotKey used for opening the HoverCard when tabbed to target.
-   * @default 'KeyCodes.c'
+   * Custom styles for this component
    */
-  openHotKey?: KeyCodes;
+  styles?: IStyleFunctionOrObject<IHoverCardStyleProps, IHoverCardStyles>;
+
+  /**
+   * Optional target element to tag hover card on.
+   * If not provided and using HoverCard as a wrapper, don't set 'data-is-focusable=true' attribute to the root of the wrapped child.
+   * When no target given, HoverCard will use it's root as a target and become the focusable element with a focus listener attached to it.
+   */
+  target?: HTMLElement | string;
+
+  /**
+   * Theme provided by higher order component.
+   */
+  theme?: ITheme;
+
+  /**
+   * Set to true if you want to render the content of the HoverCard in a FocusTrapZone for accessibility reasons.
+   * Optionally 'setInitialFocus' prop can be set to true to move focus inside the FocusTrapZone.
+   */
+  trapFocus?: boolean;
+
+  /**
+   * Type of the hover card to render.
+   * @default HoverCardType.expanding
+   */
+  type?: HoverCardType;
+}
+
+export enum OpenCardMode {
+  /**
+   * Open card by hover
+   */
+  hover = 0,
+
+  /**
+   * Open card by hot key
+   */
+  hotKey = 1
+}
+
+export enum HoverCardType {
+  /**
+   * Plain card consisting of one part responsive to the size of content.
+   */
+  plain = 'PlainCard',
+
+  /**
+   * File card consisting of two parts: compact and expanded. Has some default sizes if not specified.
+   */
+  expanding = 'ExpandingCard'
```

Comment:
For some reason in documentation I only see first enum

Reply:
This is due to the documentation parser we have not playing well with enums having string values. If I would have used 1 and 2 as values it would show up both. I actually wanted to file an issue regarding this and a couple more things I noticed with the parser. Don't think we should change it to use numbers as it's more helpful through the intellisense as it is right now.